libcxx/test/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ add_custom_target(install-cxx-test-suite-prefix
                               cxx
                               cxx_experimental
                               cxx-modules
+                              check_output
```

Comment:
You should instead do `add_dependencies(cxx-test-depends check_output)` in the place where you define `check_output` (`libcxx/test/tools/check_output/CMakeLists.txt`).